stan/math/prim/scal/prob/neg_binomial_2_log.hpp

```diff
@@ -95,6 +96,7 @@ namespace stan {
       for (size_t i = 0; i < len_np; ++i)
         n_plus_phi[i] = n_vec[i] + phi__[i];
 
+      using std::cout;
```

Comment:
We can't use `cout` or `cin` or `cerr`---it makes it impossible to wrap with R, Python or any other process that controls its own I/O.